packages/core/src/actors/promise.ts

```diff
@@ -88,8 +88,11 @@ export function fromPromise<TOutput, TInput = NonReducibleUnknown>(
      * The parent actor of the promise actor
      */
     self: PromiseActorRef<TOutput>;
+    signal: AbortSignal;
   }) => PromiseLike<TOutput>
 ): PromiseActorLogic<TOutput, TInput> {
+  const controller = new AbortController();
```

Comment:
`AbortController` is only supported since node 15. I think it's a reasonable threshold. XState v5 was released when node 18 was already the LTS